pkg/reporter/json_reporter.go

```diff
@@ -0,0 +1,68 @@
+package reporter
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type JsonReporter struct{}
+
+type fileStatus struct {
+	Path   string `json:"path"`
+	Status string `json:"status"`
+	Error  string `json:"error,omitempty"`
+}
+
+type summary struct {
+	Passed int `json:"passed"`
+	Failed int `json:"failed"`
+}
+
+type reportJSON struct {
+	Files   []fileStatus `json:"files"`
+	Summary summary      `json:"summary"`
+}
+
+// Print implements the Reporter interface by outputting
+// the report content to stdout as JSON
+func (jr JsonReporter) Print(reports []Report) error {
+	var reportJSON reportJSON
```

Comment:
Refrain from making the variable name the same name as the type name.  For example, you could use `report` for the variable name instead of `reportJSON`.